### PR TITLE
Making owner required

### PR DESCRIPTION
**Pull request by ycombinator**
This PR makes the `owner` root-level field in a package's root-level `manifest.yml` file required. 

It does **not** add any semantic validation of the value of the field, such as checking if the value corresponds to an actual owner entity in some system. This may not be possible or might require certain security privileges so let's tackle this in a follow up PR.

Resolves partially #20.

### CODE REVIEW

File: versions/1/manifest.spec.yml [hunk 1]
```diff
@@ -233,4 +233,5 @@ spec:
   - title
   - description
   - version
-  - release
\ No newline at end of file
+  - release
+  - owner
```
Comment (ruflin) [inline versions/1/manifest.spec.yml:237]: Shouldn't this be `owner.github` as this is the only property we currently use? https://github.com/elastic/package-storage/blob/production/packages/apache/0.1.4/manifest.yml#L49

Comment (mtojek) [reply versions/1/manifest.spec.yml:237]: +1 

same question here

Comment (ycombinator) [reply versions/1/manifest.spec.yml:237]: Yup, good catch! Will fix, thanks!

Comment (ycombinator) [reply versions/1/manifest.spec.yml:239]: Fixed in 57f1dcc.

**Pull request merged**